.rat-excludes

```diff
@@ -35,6 +35,7 @@ dagre-d3.min.js
 graphlib-dot.min.js
 sorttable.js
 vis.min.js
+vis.js
```

Comment:
I think it's just for debuging. See https://developer.chrome.com/devtools/docs/javascript-debugging